lib/ProductOpener/Display.pm

```diff
@@ -9040,15 +9038,17 @@ sub data_to_display_nutriscore_and_nutrient_levels($) {
 	# The Nutri-Score is unknown
 	else  {
 
-		$result_data_ref->{nutriscore_grade} = "unknown";
-
 		# Category without Nutri-Score: baby food, alcoholic beverages etc.
 		if (has_tag($product_ref,"misc","en:nutriscore-not-applicable")) {
 				push @nutriscore_warnings, lang("nutriscore_not_applicable");
+				$result_data_ref->{nutriscore_grade} = "not-applicable";
 				$result_data_ref->{nutriscore_unknown_reason} = "not_applicable";
 				$result_data_ref->{nutriscore_unknown_reason_short} = lang("nutriscore_not_applicable_short");
 		}		
 		else {
+
+			$result_data_ref->{nutriscore_grade} = "unknown";
+
```

Comment:
General bug:
On /nutrition-grades facet page, the graphic does not correspond to the facets that are underneath (the not-applicable does not appears)

Reply:
It is working for me, I see the image:

![image](https://user-images.githubusercontent.com/8158668/149097097-c9235f83-28ad-4434-944b-d957692f752e.png)

Note that the SVG images need to be rebuilt (optimized) with "make up".

Reply:
Sorry, I speak about the **chart** above the list.

Reply:
Got it! Fixed.